app/src/main/java/net/aiscope/gdd_app/ui/metadata/SampleImagesAdapter.kt

```diff
@@ -90,15 +107,17 @@ private class ImageViewHolder(
         imageFile = image
         maskFile = mask
 
+        val img: ImageView = itemView.findViewById(R.id.sample_image)
```

Comment:
Let's try to use [View Binding](https://developer.android.com/topic/libraries/view-binding) instead of `findViewById`. Also, this should be done only once when the view holder is instantiated, not on each `bind`, as the layout is not going to chance ;)